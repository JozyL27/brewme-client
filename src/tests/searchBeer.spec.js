import React from 'react'
import ReactDOM, { render } from 'react-dom'
import renderer from 'react-test-renderer'
import SearchBeer from '../components/searchBeers/searchBeer'


describe(`search beer component`, () => {

    it(`search beer component renders without crashing`, () => {
        const div = document.createElement('div')
        ReactDOM.render(<SearchBeer />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it(`renders the UI as expected`, () => {
        const tree = renderer
            .create(<SearchBeer />)
            .toJSON()
            expect(tree).toMatchSnapshot()
    })
})