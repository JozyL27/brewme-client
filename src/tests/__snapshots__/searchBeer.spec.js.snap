// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search beer component renders the UI as expected 1`] = `
<section
  className="searchSection"
>
  <h3
    className="searchHeader"
  >
    Search Beer Database
  </h3>
  <form
    className="searchDatabase"
    onSubmit={[Function]}
  >
    <input
      className="search"
      name="search"
      placeholder="Beer Database"
      type="text"
    />
    <input
      className="searchButton"
      type="submit"
    />
  </form>
  <ul
    aria-live="polite"
    className="searchUl"
  />
</section>
`;
